modules/zed/testbeds/zed.nix

```
@@ -0,0 +1,16 @@
+{ lib, pkgs, ... }:
+{
+  stylix.testbed.ui.command = {
+    text = "${lib.getExe pkgs.zed-editor} example.md";
+  };
+
+  home-manager.sharedModules = lib.singleton {
+    programs.zed-editor.enable = true;
+    home.file."example.md" = {
+      source = pkgs.fetchurl {
+        url = "https://raw.githubusercontent.com/sharkdp/bat/refs/heads/master/tests/syntax-tests/source/Rust/output.rs";
```

this should be from master, cause it might change
```suggestion
        url = "https://raw.githubusercontent.com/sharkdp/bat/e2aa4bc33cca785cab8bdadffc58a4a30b245854/tests/syntax-tests/source/Rust/output.rs";
```